### DISCUSSION TITLE
TTGO ESP32-S3 / LILYGO® T-Beam SUPREME ESP32-S3 (433Mhz)

### DISCUSSION THREAD

**Discussion by LoickFahl** [dl9rdz/rdz_ttgo_sonde#415] [TTGO firmware]
Hello ,  TTGO Team,

 I recently purchased the LILYGO® T-Beam SUPREME ESP32-S3 (433Mhz) board and have been trying to use it with the latest bin file. However, it appears that there might be compatibility issues.

Before jumping to conclusions, I wanted to reach out to inquire if there are any plans to officially support the T-Beam SUPREME ESP32-S3 in the future. If not, is there a recommended workaround or any community-driven efforts underway to address this compatibility concern?

I appreciate your time and assistance in clarifying this matter. Thank you for your ongoing efforts to support the community and enhance the functionality of TTGO products.

Thanks

**Comment by dl9rdz** [comment]
The S3 is not bin compatible, but that would be a simple issue to resolve. I could easily create pre-compiled files, or you can compile yourself a suitable bin after selecting the right architecture (S3) in ArduinoIDE or platformio (there are very minor changes to do that I have tested myself already).

The bigger issue with the T-Beam SUPREME is that it uses a different Lora chip (SX1262 instead of the older SX1278).  This is a completely different chip. I was able to make it transmit on 400 MHz, but tuning it to 400 MHz was giving rather unstable results (this frequency is outside the chip spec). Unless I find much more time for that, I am likely to give up on this. It is not very likely (in my opinion) that using the SX1262 for radiosonde reception on 400 MHz is feasible. 

I currently have some different ideas for a low-cost ESP-based radiosonde receiver, in case that SX1278 based boards will not be available any more in the future...

**Reply by LoickFahl** [reply]
Thank you so much for the detailed information. I appreciate your efforts in exploring the compatibility issues with the T-Beam SUPREME.

Regarding the .bin file, I'd be more than happy to help test it, even if it's unstable at this point. If you could provide a pre-compiled file or guide me on compiling one myself for the S3 architecture, that would be fantastic.

I understand the challenges with the T-Beam SUPREME and the use of the SX1262 Lora chip. It's great to know that you've already made progress in making it transmit on 400 MHz, even though stability remains a concern. I totally respect your decision, and I'm grateful for your honest assessment.


Thanks again for your support and expertise!

**Comment by LoickFahl** [comment]
Been trying to make it work with a new install of the Arduino IDE (2.2.1)
with the setting as selected in the picture below, but no luck so far 



E (301) esp_core_dump_flash: Core dump flash config is corrupted! CRC=0x7bd5c66f instead of 0x0
Rebooting...
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0xc (RTC_SW_CPU_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)

SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x3ac
load:0x403c9700,len:0x9a8
load:0x403cc700,len:0x2920
entry 0x403c98b8
E (254) opi psram: PSRAM ID read error: 0x00000000, PSRAM chip not found or not supported, or wrong PSRAM line mode
E (254) spiram: SPI RAM enabled but initialization failed. Bailing out.
E (273) spi_flash: Detected size(8192k) smaller than the size in the binary image header(32768k). Probe failed.
Backtrace: 0x4037822e:0x3fceb180 0x4037e1b1:0x3fceb1a0 0x40384795:0x3fceb1c0 0x420452a3:0x3fceb2f0 0x40377b9f:0x3fceb320 0x403cd6db:0x3fceb350 0x403cd99a:0x3fceb380 0x403c990d:0x3fceb4b0 0x40045c01:0x3fceb570 |<-CORRUPTED
![1](https://github.com/dl9rdz/rdz_ttgo_sonde/assets/151224621/1f689cb2-e044-4fd1-86e9-eb14e54b5e49)

**Comment by dl9rdz** [comment]
I don't have a Beam Supreme (only a T-Display S3 board and another different one). 

The flash should support QIO mode, and the flash size is 8MB (but I don't think using just DIO (which is slower) and 4MB in the config would break anything). 
(Edit: But this here "Detected size(8192k) smaller than the size in the binary image header(32768k). " is bad. It looks like you flashed the partition table or something like that for a 32MB flash. )

PSRAM most likely is QSPI, not OPI, this might explain the error regarding PSRAM.

Apart from that, I wonder how you flash the file system (the software might not run properly without). ArduinoIDE v2 does not support file system upload.  

(In any case, there will still be the SX1262 problem...)

**Comment by au51gtm** [comment]
Hello Hansi,

It would be greatly appreciated if you would update the 'Supported Boards' Wiki to specifically state which boards are supported, given the newer hardware releases.   At the moment, for example it mentions T-Beam 0.7 and 1.0 families, which are no longer available, with no mention of the 1.1, 1.2 families of Lilygo T-Beam boards, as well as the support for PMU IC changes.
This update would reduce the chance of purchasing a board which is not compatible with your software, as it stands at the moment.

Thanks and kind regards.

**Comment by au51gtm** [comment]
Hello Hansi,
I see that you have updated the 'Supported Boards' Wiki.  Thank you, this is very much appreciated.

Thanks and kind regards.
